platform/cri-o/build.patch

```diff
@@ -162,9 +162,5 @@ index 3a0a989..9e0efbe 100644
      importmap = "github.com/cri-o/cri-o/vendor/golang.org/x/crypto/ed25519",
      importpath = "golang.org/x/crypto/ed25519",
      visibility = ["//visibility:public"],
--    deps = [
--        "//vendor/crypto/ed25519:go_default_library",
--        "//vendor/golang.org/x/crypto/ed25519/internal/edwards25519:go_default_library",
--    ],
-+    deps = ["//vendor/golang.org/x/crypto/ed25519/internal/edwards25519:go_default_library"],
```

Comment:
I'm curious why this change is part of this commit. Could this perhaps be explained in the commit message?

Reply:
Well, it seems that gazelle started doing what we want; I'm not sure why.  Note that my change replaces `change X to Y` with `leave Y unchanged`; i.e. gazelle has stopped incorrectly producing `X` and is now correctly producing `Y` instead, so we don't need to do anything.

Reply:
Hmm, okay. That makes sense.